pages/forms.md

```diff
@@ -100,8 +100,6 @@ Making forms accessible is a simple process. Each form element should be associa
 
 > ___Failure:___ First name label ```for``` and ```id``` don't match.
 
-> ___Failure:___ Last name has an invalid ```id```.
```

Comment:
I removed this because it is not an accurate statement. 

`id`s starting with numbers is OK [per the W3C HTML5 spec](https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#the-id-attribute). `ID`s must be at least one character, and they cannot have spaces. Numbers are fine in any part of an `id`, though.